core/pom.xml

```diff
@@ -20,7 +20,7 @@
     <spark.version>3.2.0</spark.version>
     <fasterxml.jackson.version>2.12.2</fasterxml.jackson.version>
     <scala.binary.version>2.12</scala.binary.version>
-    <ray.version>1.9.0</ray.version>
+    <ray.version>1.13.0</ray.version>
```

Comment:
Does this have to upgraded? Is it possible to support from 1.9.0 to 2.0.0?

Reply:
2.0.0 jar is not released yet. Do you mean we just use 1.9.0 to compile?

Reply:
yes, will it work?